loader/log.c

```diff
@@ -34,14 +34,13 @@
 #include "debug_utils.h"
 #include "get_environment.h"
 
-uint32_t g_loader_debug = 0;
+uint32_t g_loader_debug = VULKAN_LOADER_ERROR_BIT;
```

Comment:
NOTE: I have now changed errors to log out to the stderr by default.  This is a change I think is important because errors currently result in no direct message from the loader other than the vague Vulkan enum values.

Reply:
Undid this for now.